src/main/java/persistence/entity/loader/LoaderMapper.java

```diff
@@ -0,0 +1,126 @@
+package persistence.entity.loader;
+
+import persistence.entity.attribute.EntityAttribute;
+import persistence.entity.attribute.EntityAttributes;
+import persistence.entity.attribute.GeneralAttribute;
+import persistence.entity.attribute.OneToManyField;
+import persistence.entity.attribute.id.IdAttribute;
+
+import java.lang.reflect.Constructor;
+import java.sql.ResultSet;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LoaderMapper {
+    private final EntityAttributes entityAttributes;
+    private final List<CollectionMapperResolver> COLLECTION_MAPPER_RESOLVERS = new ArrayList<>();
+
+    public LoaderMapper(EntityAttributes entityAttributes, CollectionLoader collectionLoader) {
+        COLLECTION_MAPPER_RESOLVERS.add(new EagerLoadingOneToManyFieldMapper(this, entityAttributes));
+        COLLECTION_MAPPER_RESOLVERS.add(new LazyLoadingOneToManyFieldMapper(entityAttributes, collectionLoader));
+
+        this.entityAttributes = entityAttributes;
+    }
+
+    public <T> T mapResultSetToEntity(Class<T> clazz, ResultSet resultSet) {
+        try {
+            EntityAttribute entityAttribute = entityAttributes.findEntityAttribute(clazz);
+            T instance = instantiateClass(clazz);
+
+            if (!resultSet.next()) {
+                return null;
+            }
```

Comment:
메서드의 검증에 대한 부분은 가장 먼저 실행 되어도 좋을 것 같은데 어떻게 생각하시나요? 😄 

```suggestion
            if (!resultSet.next()) {
                return null;
            }
            
            EntityAttribute entityAttribute = entityAttributes.findEntityAttribute(clazz);
            T instance = instantiateClass(clazz);
```